lambdas/process-journey-step/src/main/resources/statemachine/production-statemachine-config.yaml

```diff
@@ -6,10 +6,10 @@ CRI_STATE:
     next:
       type: basic
       name: next
-      targetState: EVALUATE_GPG45_SCORES
+      targetState: RETRIEVE_CRI_CREDENTIAL
       response:
         type: journey
-        journeyStepId: /journey/evaluate-gpg45-scores
+        journeyStepId: /journey/cri/credential
```

Comment:
Should we not be going from a CRI_STATE to the `/journey/cri/access-token` endpoint, and then from there to the `/journey/cri/credential` endpoint? Without collecting the access token we won't be able to get the credential? Or have I misunderstood how it works?

Reply:
@dannyhayes-gds - Can correct me if I'm wrong, but core-front calls the `/journey/cri/access-token` before it makes the call to the Journey Lambda. So the`/journey/cri/access-token` isn't actually part of the Journey States.

Reply:
Yep at the moment the front-end still calls the access-token lambda same way that used. The user lands back in core-front with an auth code so it just passes that into the access-token lambda to be used, then reverts back here for the next step.